public/index.html

```diff
@@ -7,15 +7,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1" />
   <meta name="theme-color" content="#000000" />
   <meta name="description" content="Web site created using create-react-app" />
-  <!--
-      Notice the use of %PUBLIC_URL% in the tags above.
-      It will be replaced with the URL of the `public` folder during the build.
-      Only files inside the `public` folder can be referenced from the HTML.
+  <meta property="og:title" content="Bookstore-CMS" />
```

Comment:
- [ ] While your project is beautiful as it renders now (it is!), we need to match exactly each font styling for each element on the Zeplin.

## Explanation

Look:

Your Implementation | Zeplin Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/160939130-37a6b197-ed97-4751-985f-ee1548b6bd74.png) | ![image](https://user-images.githubusercontent.com/57421823/160939084-7bfadecb-580b-46ec-a788-3663a1fc6f3f.png)

The font stylings in your project do not match the Zeplin.

We need to make your design an exact copy of the Zeplin design, this includes the font stylings.  The font stylings include: font-family, font-weight, line-height, letter-spacing, font-style, font-size, font-stretch, color.



## Hints

- Look:

![image](https://user-images.githubusercontent.com/57421823/160937917-1b948983-47f3-45ec-8380-ae81913a1366.png)

- You will need to go through each element on the Zeplin and compare it to your implementation.  If it does not match, correct it as needed.

Look:

![image](https://user-images.githubusercontent.com/57421823/160938646-53ddb498-4252-4d6c-9444-e177804ae0df.png)


- You will need to import the RobotoSlab and Montserrat font families, each with the needed weights so they render correctly.

- I understand you used Tailwind for this project.  That framework should have the option to override its stylings with yours.

- This is a CSS project, please keep in mind the next reviewer will most likely check for CSS best practices on your overrides.

_Note:  These are just tips and examples intended to help.  The main requirement is to match your font stylings exactly as the Zeplin, as requested above._

That way, your project will look exactly like the Zeplin.

Reply:
Hi @Meltrust 

Thank you for the feedback 
As always, your reviews are very thorough. I appreciate that :+1:
I'll make sure to match the design as you requested. At least for large screens.

Best,